src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1000,6 +1002,18 @@ private CaseDTO createNewCachedCaseResponse(CachedCase newCase) throws CTPExcept
     return response;
   }
 
+  private void checkCaseTypeForNewAddress(
```

Comment:
A better name which more accurately reflects what this method actually does would be:

rejectIfCEInNI()

The method does after all check then reject. As it is the name implies just check. And do what? make a note? log it?

It also ties the use of this method to only the use case of NewAddress.

Reply:
yes good point - I wasn't sure what to call it - I shall change it